back/node_watcher/src/tasks/createStake.ts

```diff
@@ -18,7 +18,11 @@ const l = logger('Task: Stake');
  */
 const createStake: Task<NomidotStake> = {
   name: 'createStake',
-  read: async (blockHash: Hash, api: ApiPromise): Promise<NomidotStake> => {
+  read: async (
+    blockHash: Hash,
+    _cached: Cached,
+    api: ApiPromise
+  ): Promise<NomidotStake> => {
     const currentElected = await api.query.staking.currentElected.at(blockHash);
     const stakersInfoForEachCurrentElectedValidator: Exposure[] = [];
     let totalStaked = new BN(0);
```

Comment:
unrelated to this PR: i would advocate to never use BN directly, always use `api.createType`